packages/form/src/LionForm.js

```diff
@@ -18,16 +18,23 @@ export class LionForm extends DelegateMixin(LionFieldset) {
     };
   }
 
+  constructor() {
+    super();
+    this.__boundSubmit = this._submit.bind(this);
+    this.__boundReset = this._reset.bind(this);
+  }
+
   connectedCallback() {
     super.connectedCallback();
-    this.addEventListener('submit', this._submit);
-    this.addEventListener('reset', this._reset);
+
+    this.addEventListener('submit', this.__boundSubmit);
+    this.addEventListener('reset', this.__boundReset);
   }
 
   disconnectedCallback() {
     super.disconnectedCallback();
-    this.removeEventListener('submit', this._submit);
-    this.removeEventListener('reset', this._reset);
+    this.removeEventListener('submit', this.__boundSubmit);
+    this.removeEventListener('reset', this.__boundReset);
```

Comment:
let's be more consistent :)

![image](https://user-images.githubusercontent.com/137844/57769085-53757e80-770d-11e9-867b-12e7e083ba89.png)

Reply:
empty line removed